virttest/utils_libvirt/libvirt_monitor.py

```diff
@@ -8,30 +8,107 @@
 LOG = logging.getLogger('avocado.' + __name__)
 
 
-def check_domjobinfo(vm_name, expected_dict, remote=False, remote_ip=None, options=""):
+def check_domjobinfo_output(params):
     """
-    Check domjobinfo
+    Check domjobinfo output
 
-    :param vm_name: vm name
-    :param expected_dict: the dict for expected domain job info
-    :param remote: check remote host if remote is True
-    :param remote_ip: the ip of remote host
-    :param options: the options of domjobinfo command
+    :param params: Dictionary with the test parameters
     """
-    LOG.info("Check domjobinfo")
-    if remote:
+    def check_domjobinfo_items(expected_domjobinfo):
+        """
+        Check the items of domjobinfo output
+
+        :param expected_domjobinfo: the expected domjobinfo
+        """
+        def compare_value(value_type, line):
```

Comment:
compare_value and check_items container long codes. Could they be moved outside check_domjobinfo_output. 
If those function are exclusively for check_domjobinfo_output's use, we can use _ prefix to define them as inner methods

Reply:
Updated, thanks.